### PR TITLE
Fix: #1281 null value displayed in Client Activity toolbar

### LINKED ISSUE #1281: Null value displayed in Client Activity toolbar
**Summary:** 

Null value is displayed in Client Activity toolbar if last name of the client does not exist

**Steps to reproduce:** 

1. Nav Bar -> Clients
2. Click on a client for which surname does not exist
3. Null display on the toolbar

**Expected behaviour:** 

In this case first name should be displayed

**Observed behaviour:** 

Instead, null value is displayed

**Device and Android version:** 

Android 10
OnePlus 6
Stock ROM

**Screenshots:** 

null client name

### PR DESCRIPTION
**Pull request by Grandolf49**
Fixes #1281 #1282 
Both issues are solved in this pull request

Please Add Screenshots If there are any UI changes.

fix - null value in client activity

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by laxyapahuja** [comment]
hey @Grandolf49. Please look into your Travis CI Build. You have some PMD violations.

**Comment by Grandolf49** [comment]
@laxyapahuja Yes, I had PMD violation. Didn't put braces after if-else statement as it was single line code.
Have updatedthe PR

**Comment by laxyapahuja** [comment]
@Grandolf49 Please squash your commits.

**Comment by Grandolf49** [comment]
@laxyapahuja Squashed the commits and updated the PR!

File: mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml [hunk 1]
```diff
@@ -19,14 +19,15 @@
 
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
```
Comment (luckyman20) [inline mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml]: Changes in this file are not related to this issue.

Comment (Grandolf49) [reply mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml]: @luckyman20 Actually  #1281 #1282
Both issues are solved in this pull request. There was a small issue so I squashed the commits.
Should I create 2 separate PRs for the same?

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 2]
```diff
@@ -41,6 +35,12 @@
 import android.widget.TextView;
 import android.widget.Toast;
 
+import androidx.appcompat.widget.PopupMenu;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
+import androidx.core.content.res.ResourcesCompat;
+import androidx.fragment.app.FragmentTransaction;
+
```
Comment (robustTechie) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Reordering is of no use here

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 3]
```diff
@@ -245,7 +245,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 // Get the Image from data
 
                 Uri selectedImage = data.getData();
-                String[] filePathColumn = { MediaStore.Images.Media.DATA };
+                String[] filePathColumn = {MediaStore.Images.Media.DATA};
```
Comment (robustTechie) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Here removing space is not needed I think.

File: mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml (hunk 1 above)
Comment (elirehema) [reply mifosng-android/src/main/res/layout/fragment_savings_account_summary.xml]: > @luckyman20 Actually #1281 #1282
> Both issues are solved in this pull request. There was a small issue so I squashed the commits.
> Should I create 2 separate PRs for the same?

You need to make single PR for single issue.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java (hunk 2 above)
Comment (Grandolf49) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: It's the result of auto re-factoring the code in Android Studio

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java (hunk 3 above)
Comment (Grandolf49) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Same for this too

**Comment by robustTechie** [comment]
#1324 is the same fix so please close the duplicate PR
